core/services/ardupilot_manager/main.py

```diff
@@ -16,6 +20,17 @@ def __init__(self) -> None:
         if not available_mavlink_interfaces:
             raise RuntimeError("No MAVLink interface available.")
 
+        self.settings_path = appdirs.user_config_dir(self.__class__.__name__)
```

Comment:
does this end up being /root/.config/ArdupilotManager?

Reply:
Yes.